tests/python/pytorch/graphbolt/impl/test_disk_based_feature_store.py

```diff
@@ -64,31 +65,16 @@ def test_disk_based_feature():
         with pytest.raises(IndexError):
             feature_a.read(torch.tensor([0, 1, 2, 3]))
 
+        with pytest.raises(AssertionError):
+            assert not np.isfortran(
```

Comment:
the testcase is expected to capture the exception when instantiating `DiskBasedFeature` instead of check if isfortran.

Reply:
please capture the error message as well

Reply:
refer to https://github.com/dmlc/dgl/blob/41a38486a5ed9298093d9f0bc415751269c7d577/tests/python/pytorch/graphbolt/test_itemset.py#L33-L37